operating/manage-compliance.adoc

```diff
@@ -0,0 +1,74 @@
+[id="manage-compliance"]
+= Managing compliance
+include::modules/common-attributes.adoc[]
+:context: manage-compliance
+
+toc::[]
+
+[role="_abstract"]
+By using {product-title} you can assess, check, and report on the compliance status of your containerized infrastructure.
+You can run out-of-the-box compliance scans based on industry standards including:
+
+* *CIS Benchmarks* (Center for Internet Security) for *Docker* and *Kubernetes*,
```

Comment:
I'd remove the commas, the period, the "and", etc. after each of these bullet items